src/variable.test.js

```diff
@@ -0,0 +1,46 @@
+import isWindowsMock from 'is-windows'
+import varValueConvert from './variable'
+
+beforeEach(() => {
+  isWindowsMock.__mock.reset()
+})
+
+test(`doesn't affect simple variable values`, () => {
+  isWindowsMock.__mock.returnValue = true
+  expect(varValueConvert('foo')).toBe('foo')
+})
+
+test(`converts a : into a ; on Windows`, () => {
+  isWindowsMock.__mock.returnValue = true
+  expect(varValueConvert('foo:bar')).toBe('foo;bar')
+})
+
+test(`converts a ; into a : on UNIX`, () => {
```

Comment:
Actually, could we simplify things a little bit? @hgwood and I have decided that it would be nicer if `cross-env` allowed you to _only_ use the unix notation for things and not convert the other way. That would make the implementation simpler and the API simpler. Could you back these things out and even add a test or two to make sure that `;` is _not_ converted whether on unix or not? Thanks!